docs/eventing/samples/apache-camel-source/README.md

```diff
@@ -0,0 +1,94 @@
+These samples show how to configure a Camel source. It is a event source that
+can leverage one of the [250+ Apache Camel components](https://github.com/apache/camel/tree/master/components)
+for generating events.
+
+## Prerequisites
+
+1. Setup [Knative Serving](../../../serving).
```

Comment:
Suggestion: link to the main install page instead and say "Install Knative Serving and Eventing" as one step.